frontend/src/scenes/trends/PersonModal.tsx

```diff
@@ -61,8 +90,8 @@ export function PersonModal({ visible, view, filters, onSaveCohort }: PersonModa
         [filters, people, isInitialLoad]
     )
 
-    const isDownloadCsvAvailable = view === ViewType.TRENDS
-    const isSaveAsCohortAvailable = clickhouseFeaturesEnabled
+    const isDownloadCsvAvailable = view === ViewType.TRENDS || view === ViewType.STICKINESS
+    const isSaveAsCohortAvailable = clickhouseFeaturesEnabled && isDownloadCsvAvailable
```

Comment:
This removes the "Save as Cohort" button from the funnel persons modal because it didn't work.
@EDsCODE why keep two variables though?

Reply:
thought it was used elsewhere for some reason. just removed it